docs/learn/tokenomics2/Inflation.md

```diff
@@ -2,33 +2,166 @@
+### Cycles
+**Cycle** can be considered as a 'year' in the Astar Network.
+When a new cycle starts, new inflation configuration is calculated according to the total issuance at that point in time.
+E.g. if 'yearly' inflation is set to be 7%, and total issuance is 1,000,000 ASTR, then the soft-capped max inflation for that
+cycle will be 70,000 ASTR.
 
-### Treasury rewards
+Cycle length is configurable, but in practice it will resemble a 'year' off-chain.
 
-The dynamic treasury allocation has been removed in favor of a fixed annual inflation rate of 5%. Which correspond to a reward of 11.06 ASTR per block.
+ $inflation\_soft\_cap = inflation\_rate * total\_issuance$
 
-### Collators rewards
+### Periods
 
-Collators will now receive a steady 3.2% of the annual inflation which correspond to a reward of 7.07 ASTR per block. Although this represents a reduction from the previous model, the upcoming alignment of EVM fees and Substrate native fees is expected to increase overall fee earnings for collators.
+Each **cycle** consists of one or more **periods**.
+**Periods** are a core time unit in the dApp Staking protocol. Each period consists of a `Voting` and `Build&Earn` subperiods.
 
-### Stakers & dApp Rewards
+During the `Voting` subperiod, neither stakers nor dApp owners can earn any rewards, only _collators_ and _treasury_ continue earning.
+During the the `Build&Earn` subperiod, stakers earn staking rewards per **era** and have the opportunity to earn bonus reward at the end of a period.
+dApp owners also earn rewards, based on how well their dApp is performing in dApp staking, at the end of each **era**.
 
-The rewards for dApp staking, both for users and dApps, will remain unchanged in absolute terms. No immediate modifications will be implemented in this area. Future updates, including the introduction of dApp staking v3, will occur in the third phase of Tokenomics 2.0.
+### Eras
 
-We use the blockReward distribution to ensure those three requirements (and calculate them from the reward per block):
+Each **period** is divided up into multiple **eras**.
+**Era** is the core time unit in dApp staking, and its length is measured in the number of blocks.
 
-|                  | Percentage | Reward ASTR |
-|------------------|------------|-------------|
-| Block reward     | 100%       | 231.20      |
-| Treasury         | 4.78%      | 11.06       |
-| Collators        | 3.06%      | 7.07        |
-| dApp reward      | 17.27%     | 39.93       |
-| Base Staker      | 23.04%     | 53.27       |
-| Ajustable Staker | 51.84%     | 119.85      |
\ No newline at end of file
+`BuildAndEarn` subperiod consists of one or more **eras**. Each era has a fixed length.
+`Voting` subperiod always consists of exactly **one era** but its length can be longer than a standard era length.
```

Comment:
This part is confusing. It is not incorrect, but still thinking how to write it simpler.

Why can't you write (for the sake of better understanding):

`Voting` subperiod always consists of exactly **one era** and this 'voting era' is unique as its length can be longer than a standard era length (it has more blocks than standard era).

Reply:
I don't have anything against changing how it's written but you should also emphasize that the length is of the `Voting` subperiod is a multiple of standard era length.

Reply:
I adjusted please check

Reply:
I'm ok with it if it's more clear for you.